### DISCUSSION TITLE
Welcome to share-me Discussions!

### DISCUSSION THREAD

**Discussion by Dan6erbond** [Dan6erbond/share-me#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Candyxxy** [comment]
How do i keep updated on the latest versions?

**Reply by Candyxxy** [reply]
Im very unexperienced, please dont blame :)

**Reply by Dan6erbond** [reply]
Hey! If you're deploying with Docker compose just make sure you're using the `latest` tag or no tag and when an update is released you need to restart the container. 

Since the app only relies on the files in the `pb_data` folder as long as you have a volume configured you can just restart it and it should work.

**Reply by Candyxxy** [reply]
So i just remove the both version numbers from server and frontend image to "latest"?
(image: ghcr.io/dan6erbond/share-me-server:latest)
(image: ghcr.io/dan6erbond/share-me-frontend:latest)

Thanks for your quick response :)

**Reply by Dan6erbond** [reply]
Yes exactly. That should do the trick, although keep in mind that the reason the version was specified is because Share Me is still an early product. So breaking changes could be introduced that you might not want auto-updated.
